### PR TITLE
cjose 0.6.1 (new formula)

### PR DESCRIPTION
**Pull request by eoger**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/cjose.rb [hunk 1]
```diff
@@ -0,0 +1,32 @@
+class Cjose < Formula
+  desc "C library implementing Javascript Object Signing and Encryption (JOSE)"
```
Comment (ilovezfs) [inline Formula/cjose.rb]: The description for the `jose` formula is

```
  desc "C-language implementation of Javascript Object Signing and Encryption"
```

Is there anything that distinguishes the cjose implementation?

Comment (eoger) [reply Formula/cjose.rb]: I think these are just different implementations, this one by Cisco.

**Comment by eoger** [comment]
Bump, let me know if you need more changes :)

**Comment by thomcc** [comment]
Was there a reason this wasn't merged?

**Pull request closed without merging**